### ISSUE TITLE
[Visual Requirement-Project-Example]: Luminosity ratio is less than 4.5:1 for the text showing in blue color.

### ISSUE THREAD

**Issue by v-swasha** [Microsoft/pxt#2536]
**User Experience:**
Users who depends on Keyboard with Screen reader will not be able to access the any Textual content if its luminosity ratio is less than 4.5:1.

**Test Environment:**
OS: RS2 Version 1703(OS Build 15063.483)
Platform: Edge.
Tool Used: Color Contrast Analyzer

**Repro Steps:-**
1. Navigate to https://makecode.microbit.org/acc
2. Navigate to the Microbit section element and select Code control given on it.
3. Navigate to the Projects control lying in the header section on the page and select it.
4. Navigate to the Examples Tab on the page opened and Select any project.
5. Navigate to various controls lying on the page opened..
6. Verify the Luminosity ratio for the all the text showing using CCA tools.

**Actual Result:-**
Luminosity ratio is less than 4.5:1 for the text showing in blue color.

**Expected Result:-**
Luminosity ratio should be more than 4.5:1 for the text showing in blue color.

**Note:** This issue exists throughout app for text showing in blue color.

**MAS Reference -**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

**Suggested Fix:**
1.1-Ensuring that a contrast ratio of at least 4.5:1 exists between text (and images of text) and background behind the text 
1.2- If design allows , Not specify background color, not specify text color, and not using technology features that change those defaults .In this technique the author avoids having to do any contrast measures by simply not specifying the text color and not specifying the background. As a result the colors of both are completely determined by the user agent.
Tips:-
•The easiest fix is to increase the brightness of light colors and decrease the brightness of darker colors. 
•Exaggerate lightness differences between foreground and background colors, and avoid using colors of similar brightness adjacent to one another, even if they differ in saturation or hue. 
•During design Use the Color Contrast Analyzer to determine whether new colors have color contrast ratio of at least 4.5:1. 
 
Key References for Fix: 
http://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html
 
Please refer the attachment
![mas1 4 3_projecs_examples_select any project](https://user-images.githubusercontent.com/30459830/28660616-500ed6e8-72d1-11e7-9e4a-02ae296aec90.jpg)

**Comment by v-amik** [comment]
review done

**Comment by veler** [comment]
Need to be investigated with @microsoftsam

**Comment by samelhusseini** [comment]
Color meets contrast requirements for 18px. Users with issues have two options: 
- switch to High contrast 
- increase size of text (zoom in)

I'm not sure we can change the colors at this point. Do these options suffice @v-mesala ?

**Comment by sonimishra** [comment]
Bug is verified on URL: https://makecode.microbit.org/acc and issue is fixed as now luminosity ratio is more than 4.5:1 .hence closed the bug.I don't have permissions to add/Remove labels.

**Comment by samelhusseini** [comment]
Okay great, closing issue.
